bot/exts/pride/prideleader.py

```diff
@@ -0,0 +1,90 @@
+import json
+import logging
+import random
+from pathlib import Path
+from typing import Optional
+
+import discord
+from discord.ext import commands
+from fuzzywuzzy import fuzz
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+
+class PrideLeader(commands.Cog):
+    """Gives information about Pride Leaders."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.pride = self.load_pride_leader_json()
+
+    @staticmethod
+    def load_pride_leader_json() -> dict:
+        """Loads pride leader information from static json resource."""
+        explanation_file = Path("bot/resources/pride/prideleader.json")
+        with explanation_file.open(encoding="utf8") as json_data:
+            pride = json.load(json_data)
+
+        return pride
+
+    def name_verifier(self, leader_name: str) -> Optional[str]:
+        """Verify leader name whether it is present in json or not."""
+        leader_name = leader_name.title()
+        if leader_name in self.pride:
+            log.trace("Got Valid name.")
+            return leader_name
+
+    def invalid_embed_generate(self, pride_leader: str) -> discord.Embed:
+        """Generates Invalid Embed."""
+        embed = discord.Embed()
+        embed.color = Colours.soft_red
+        valid_name = []
+        pride_leader = pride_leader.title()
+        for name in self.pride:
+            if fuzz.ratio(pride_leader, name) >= 40:
+                valid_name.append(name)
+
+        if not valid_name:
+            valid_name = ", ".join(name for name in self.pride.keys())
```

Comment:
You can simply call join on the dict keys.
```suggestion
            valid_name = ", ".join(self.pride.keys())
```